app/services/email_manager/sender.rb

```diff
@@ -18,13 +18,32 @@ def self.send_receipt(to:, html:)
                       subject: 'Receipt from Send Chinatown Love',
                       html: html
 
-      # Send to Receipts Eng so that they know what their receipts in prod looks
-      # like
+      # Send to Receipts Eng so that they know what their receipts in prod looks like
       RestClient.post api_url,
                       from: 'receipts@sendchinatownlove.com',
                       to: 'receipts@sendchinatownlove.com',
                       subject: 'Receipt from Send Chinatown Love',
                       html: html
     end
+
+    def self.send_receipt_with_subject(to:, html:, subject:)
```

Comment:
I'm wondering if this is adding too much redundancy but I think this was the easiest way to send an email with a custom subject

Reply:
@justintmckibben was using this to send the gift card email with a custom subject, should we use this for the passport/food crawl redemption email?